movement/log_config.py

```diff
@@ -1,9 +1,8 @@
 import logging
 from logging.handlers import RotatingFileHandler
 
-from rich.logging import RichHandler
-
-FORMAT = "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+FORMAT = "%(asctime)s - %(levelname)s - "
```

Comment:
Just because of the linting line length limit. 
I could also use the parentheses syntax (or any number of other syntactical options for this kind of thing)
```python
foo = ("This is a very long
          "long string")
```

Reply:
I _think_ parentheses would be better because the linters would recognise this as one line and reformat as necessary. Otherwise if this line gets longer in the future, ruff might convert it to:

```python
THIS = "is a very long string"
THIS = ("is a longer string
              that the linter has split across two lines")

Reply:
Good point, I changed it to parentheses